src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
Now I remember I've seen this option when I was looking into the issue in the past. The disadvantage of this option is (provided I read the documentation correctly) that it then includes all .o files from the specific static library in the target dynamic library even if they are neither being used nor marked with the "used" attribute. 
In other words, the fine grained control provided by the "used" attribute is gone.
Having said that, it seems to me that it is a solution that we should use here.